server/webapp/WEB-INF/rails.new/app/views/admin/server/index.html.erb

```diff
@@ -93,8 +94,14 @@
                         <span id="password_file_path_error_message"></span>
                     </div>
                 </div>
+                  <%= render :partial => 'shared/form_required_message.html', :locals => {:scope => {}} %>
+
+              <% else -%>
+                  <div class="information">
+                    <span>Support for inbuilt LDAP and Password file have been deprecated in favour of the bundled plugins for the same.</span>
```

Comment:
Does this look ok?

<img width="1331" alt="screen shot 2017-05-23 at 3 49 42 pm" src="https://cloud.githubusercontent.com/assets/1046342/26350057/7bf5466e-3fcf-11e7-8d01-f84f5cff1ffe.png">